utils/RunONNXModel.py

```diff
@@ -429,6 +453,8 @@ def read_input_from_refs(num_inputs, load_ref_filename, is_load_ref):
     inputs = []
 
     if is_load_ref:
+        from onnx import numpy_helper
```

Comment:
Would there be a way to store it as numpy array and thus not rely on the onnx package at all? I am not familiar with numpy serialization, but maybe there is something already built in?